devel/readline/Portfile

```diff
@@ -35,15 +35,19 @@ if {![regexp {0*([1-9]+)} $patchlevel -> lastpatch]} {
 }
 
 # remove "-arch_only `/usr/bin/arch`"
```

Comment:
This comment should be removed too, since you removed the patchfile that did this.
```suggestion
```